Sources/StreamChat/Database/DTOs/PollVoteDTO.swift

```diff
@@ -46,7 +46,14 @@ class PollVoteDTO: NSManagedObject {
     
     static func load(voteId: String, pollId: String, context: NSManagedObjectContext) -> PollVoteDTO? {
         let request = fetchRequest(for: voteId, pollId: pollId)
-        return load(by: request, context: context).first
+        let results = load(by: request, context: context)
+        if results.count > 1 {
+            for i in 1..<results.count {
+                let dto = results[i]
+                context.delete(dto)
+            }
+        }
```

Comment:
Can you explain this one? Maybe add a comment? I get the feeling this should not be part of the `load` function, since it is doing more than just loading, should this logic be outside of the load function?

Reply:
actually removed it, found a better way